### PR TITLE
Use assertJ assertions

### LINKED ISSUE #9: Use only one library of assertions
We currently use AssertJ *and* Junit's assertions.
Using only one in our tests would be nice.

### PR DESCRIPTION
**Pull request by FaustXVI**
Closes #9 
- [x] Use only AssertJ in tests

### CODE REVIEW

File: src/test/java/com/github/junit5docker/DefaultDockerClientIT.java [hunk 1]
```diff
@@ -39,6 +39,24 @@ public void getExistingContainers() {
         containers = dockerClient.listContainersCmd().exec();
     }
 
+    @AfterEach
+    public void stopAndRemoveStartedContainers() {
+        dockerClient.listContainersCmd().exec().stream()
+                .filter(container -> !containers.contains(container))
```
Comment (vdemeester) [inline src/test/java/com/github/junit5docker/DefaultDockerClientIT.java]: We should rename `containers` to `existingContainers` or something 👼

File: src/test/java/com/github/junit5docker/DefaultDockerClientIT.java [hunk 2]
```diff
@@ -86,9 +105,8 @@ public void shouldStartContainerWithEnvironmentVariables() {
                 String containerId = defaultDockerClient.startContainer(WANTED_IMAGE, environments);
                 InspectContainerResponse startedContainer = dockerClient.inspectContainerCmd(containerId).exec();
                 List<String> envs = Arrays.asList(startedContainer.getConfig().getEnv());
-                assertEquals(2 + DEFAULT_DOCKER_ENV_NUMBER, envs.size());
-                assertTrue(envs.contains("khaled=souf"));
-                assertTrue(envs.contains("abdellah=stagiaire"));
+                assertThat(envs).hasSize(2 + DEFAULT_DOCKER_ENV_NUMBER)
+                        .contains("khaled=souf", "abdellah=stagiaire");
```
Comment (vdemeester) [inline src/test/java/com/github/junit5docker/DefaultDockerClientIT.java:109]: Didn't see *abdellah=stagiaire* ❤️
/cc @zeroual

**Review by vdemeester — APPROVED** [review]
A small comment but overall LGTM 🐮
needs a rebase though 👼

File: src/test/java/com/github/junit5docker/DefaultDockerClientIT.java (hunk 2 above)
Comment (zeroual) [reply src/test/java/com/github/junit5docker/DefaultDockerClientIT.java:109]: hahahhaha

**Pull request merged**